docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,512 @@
+node is shut down.
 
-crate-node CLI
-==============
+You can temporarily shut down nodes for several reasons, e.g., maintenance or
+systems failure. Once a node is shut down, you need to restart it to let the
+node rejoin the cluster. However, rejoining nodes isn’t always possible. For
+instance, a node could be irreparable and would not restart. With the cluster
+still available, you can replace a broken node by a new node on a new host.
+
+Modifying a node
+================
+
+. NOTE::
+
+  You can only perform the following operations after you have shut down the
+  affected node.
+
+The ``crate-node`` command enables you to adjust the role of a node and may also
+be able to recover some data after a disaster.
+
+You first have to call up the node, in order to modify it. Use these arguments:
 
 .. code-block:: console
 
    bin/crate-node repurpose|unsafe-bootstrap|detach-cluster
    [--ordinal <Integer>] [-C <KeyValuePair>]
    [-h, --help] ([-s, --silent] | [-v, --verbose])
 
+The ``crate-node`` command has three modes — repurpose, unsafe-bootstrap, and
+detach cluster:
 
-The following commands provide a various operations to modify and recover nodes:
+* Use ``crate-node repurpose`` to delete data from a node if it used to be a
+  data node or a master-eligible node but has been repurposed to have none of
+  these roles.
 
-* ``crate-node repurpose`` can be used to delete unwanted data from a node if it
-  used to be a data node or a master-eligible node but has been repurposed not to
-  have one or other of these roles.
-
-* ``crate-node unsafe-bootstrap`` can be used to perform unsafe cluster bootstrapping.
-  It forces one of the nodes to form a brand-new cluster on its own, using its local
+* Use ``crate-node unsafe-bootstrap`` to perform unsafe cluster bootstrapping.
+  It forces one of the nodes to form a new cluster on its own, using its local
   copy of the cluster metadata.
 
-* ``crate-node detach-cluster`` enables you to move nodes from one cluster to another.
-  This can be used to move nodes into a new cluster created with the
-  ``crate-node unsafe-bootstap`` command. If unsafe cluster bootstrapping was not
-  possible, it also enables you to move nodes into a brand-new cluster.
+* Use ``crate-node detach-cluster`` to move nodes from one cluster to another,
+  move nodes into a new cluster created with ``crate-node unsafe-bootstrap``,
+  and move nodes into a brand-new cluster if ``crate-node unsafe-bootstrap`` was
+  not possible.
 
 Changing the role of a node
 ---------------------------
 
-Certain situations may require to change the initial type of a node. The
-``crate-node repurpose`` command can modify and delete the data stored on a  disk
-from a node.
+Certain situations may require you to change the initial type of a node by
+modifying and deleting its on-disk data.
+
+Run ``crate-node repurpose`` on a node with ``node.data: false`` and
```

Comment:
are these config values provided on the command line? if so, reformat this so that it uses a full command-line example in a literal block

if not, explain the workflow here

Reply:
These config values can be provided as comming line argument as well as set in the crate.yml config file. Usually you would set these values in crate.yml.

Reply:
I would not format it as a command-line block since the values are usually declared in crate.yml and not as a command line argument.

Reply:
Okay, thanks for the info.